runtime/integration-tests/src/cases/lp/investments.rs

```diff
@@ -561,16 +567,20 @@ mod with_foreign_currency {
 			lp_utils::process_outbound::<T>(|msg| {
 				assert_eq!(
 					msg,
-					pallet_liquidity_pools::Message::FulfilledCancelDepositRequest {
-						pool_id: POOL_A,
-						tranche_id: pool_a_tranche_1_id::<T>(),
-						investor: vec_to_fixed_array(lp::utils::remote_account_of::<T>(
-							Keyring::TrancheInvestor(1)
-						)),
-						currency: utils::index_lp(evm, names::USDC),
-						currency_payout: DEFAULT_INVESTMENT_AMOUNT,
-						fulfilled_invest_amount: DEFAULT_INVESTMENT_AMOUNT,
-					}
+					GatewayMessage::Outbound {
+						sender: <T as pallet_liquidity_pools_gateway::Config>::Sender::get(),
+						destination: Domain::EVM(EVM_DOMAIN_CHAIN_ID),
+						message: Message::FulfilledCancelDepositRequest {
+							pool_id: POOL_A,
+							tranche_id: pool_a_tranche_1_id::<T>(),
+							investor: vec_to_fixed_array(lp::utils::remote_account_of::<T>(
+								Keyring::TrancheInvestor(1)
+							)),
+							currency: utils::index_lp(evm, names::USDC),
+							currency_payout: DEFAULT_INVESTMENT_AMOUNT,
+							fulfilled_invest_amount: DEFAULT_INVESTMENT_AMOUNT,
+						},
+					},
```

Comment:
Maybe here, the sender/destination can just checked in `process_outbound` method, leaving each test as before without that overhead

Reply:
I mean, I think for all cases the `sender` and `destination` are always the same